packages/app/src/vis-packs/nexus/containers/NxComplexImageContainer.tsx

```diff
@@ -52,9 +52,9 @@ function NxComplexImageContainer(props: VisContainerProps) {
       <DimensionMapper
         className={visualizerStyles.dimMapper}
         dims={dims}
-        axisLabels={axisLabels}
+        dimHints={axisLabels}
```

Comment:
`axisLabels` was poorly named; "dimension labels" would have made more sense, but I went with "dimension hints" to remain consistent with the naming used inside `DimensionMapper`.